week-3/debug/format-as-12-hours.js

```diff
@@ -1,6 +1,8 @@
 function formatAs12HourClock(time) {
   if (Number(time.slice(0, 2)) > 12) {
-    return `${Number(time.slice(0, 2)) - 12}:00 pm`;
+    return `${(Number(time.slice(0, 2)) - 12)
+      .toString()
+      .padStart(2, "0")}:${time.slice(time.indexOf(":") + 1)} pm`; // I  got this part
```

Comment:
Would you be able to refactor it a bit? You have same piece of code 2 times which may be stored in a variable instead: `Number(time.slice(0, 2))`.